package.json

```diff
@@ -53,6 +53,7 @@
     "react-test-renderer": "^15.6.1",
     "standard": "^10.0.2",
     "styled-components": "^2.1.1",
+    "stylis": "^3.2.16",
```

Comment:
Just a note for context here: we were having snapshot issues with an older version of this being installed due to styled-components' specified version range, and this seemed like the simplest solution to stop the snapshot's from failing on Travis